components/search_results_item/search_results_item.jsx

```diff
@@ -206,7 +206,14 @@ export default class SearchResultsItem extends React.PureComponent {
 
         let botIndicator;
         if (post.props && post.props.from_webhook) {
-            botIndicator = <div className='bot-indicator'>{Constants.BOT_NAME}</div>;
+            botIndicator = (
```

Comment:
Thanks for changing these too. I made a help wanted ticket to change those, but I can close it now